thumbs/Pipfile

```diff
@@ -6,11 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-Pillow = ">=6.0,<10.0"
+Pillow = "==10.4.0"
 boto3 = "*"
 sorl-thumbnail = "*"
 six = "*"
-django = "==2.2.27"
+django = "==4.2.15"
 
 [requires]
-python_version = "3.7"
+python_version = "3.12.5"
```

Comment:
This is going to be an annoyingly specific pin. Can we make this just 
python_version = "3.12"

Reply:
Fixed with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/fdb568fc537adc00a94dc24c9a155a9aa859d119.